test/integration/targets/dynamodb/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# defaults file for dynamodb
+aws_region: eu-west-1
+dynamo_db_table: ansible-dynamodb-int-test
```

Comment:
Use a random name to avoid conflicts with other test runs. Look at the existing AWS integration tests for examples.